### PR TITLE
mysql_connections_per_user: fix for non-alphanumeric symbols in fields

### PR DESCRIPTION
**Pull request by n0guest**
Hi.

This is small fix for a non-alphanumeric symbols in field names plus print real username in label.

### CODE REVIEW

File: plugins/mysql/mysql_connections_per_user [hunk 1]
```diff
@@ -100,6 +100,9 @@ sub print_graph_data() {
         if($print_user eq "root") {
             $print_user = "root_";
         }
+        if (  $print_user =~ /[^A-Za-z0-9]/ ) {
+            $print_user =~ s/[^A-Za-z0-9]/_/;
+        }
```
Comment (sumpfralle) [inline plugins/mysql/mysql_connections_per_user]: Please use the perl module "Munin::Plugin" for generating a proper fieldname according to munin's restrictions. Just use `clean_fieldname($print_user)`. This function also handles the "root" exception.
Thus you can avoid duplicate code.

**Comment by sumpfralle** [comment]
@n0guest: did I possibly overwhelm you with my wishlist? :)

In case you want to focus on this single issue, then I could also merge your proposal as it is. Just tell me ...

**Comment by n0guest** [comment]
> @n0guest: did I possibly overwhelm you with my wishlist? :)

Nope, you didn't. Just little overwhelmed at work within few last weeks. But i'll try to follow your requests at this week or next.

**Comment by sumpfralle** [comment]
@n0guest: just a friendly (and patient) reminder ...

**Comment by sumpfralle** [comment]
Ping?

**Comment by sumpfralle** [comment]
@n0guest: any news?

**Comment by sumpfralle** [comment]
I merged you original fix. Feel to improve the other bits of the plugins (mentioned in my first comment), if you feel like ...

**Pull request merged**